lib/core-common/src/utils/has-dotenv.ts

```diff
@@ -0,0 +1,11 @@
+import fs from 'fs';
+import path from 'path';
+
+// https://github.com/mrsteele/dotenv-webpack/blob/master/src/index.js#L34
+const DOTENV_FILE = path.join('.', '.env');
```

Comment:
Aren't some people defining special environment specific .env files which get auto-loaded?

`.prod.env` etc?

Reply:
Not in our current setup